shell/browser/extensions/extension_tab_util.cc

```diff
@@ -0,0 +1,243 @@
+#include "shell/browser/extensions/extension_tab_util.h"
+
+#include <iostream>
+#include <memory>
+#include <utility>
+
+#include "base/stl_util.h"
+#include "base/strings/pattern.h"
+#include "chrome/browser/profiles/profile.h"
```

Comment:
We try to avoid #including `profile.h` when we can.

Reply:
Oh, I didn't notice this one